PrincessBrideTrivia/PrincessBrideTrivia.Tests/ProgramTests.cs

```diff
@@ -6,6 +6,28 @@ namespace PrincessBrideTrivia.Tests
     [TestClass]
     public class ProgramTests
     {
+        
+
+        [TestMethod]
+        [DataRow("1")]
+        [DataRow("2")]
+        [DataRow("3")]
+        public void CheatCodeAlwaysReturnsTrue(string correctIndex)
```

Comment:
```suggestion
        public void DisplayResult_GivenCheatCode_ReturnsTrue(string correctIndex)
```
The readability here could be improved by following the recommended naming convention for tests specified here:
https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices

```suggestion
        public void DisplayResult_ReturnsTrueIfGivenCheatCode(string correctIndex)
```
Alternatively, you could name it according to the convention already set within the code.
Such as the existing method: "DisplayResult_ReturnsTrueIfCorrect()"
Although, I do prefer the convention specified by microsoft.

Pull Request title includes "Assignment 1" in the title ✔
Pull Request targets Assignment 1 branch ✔
Issue 1: Application no longer crashes ✔
Issue 2: Unit test properly passes ✔
Issue 3: Add a new feature request implemented ✔
Issue 3: Feature request unit tested ✔

Looks great! You should consider [slanante's](https://github.com/slanante) suggestion for GetPercentCorrect(). You could also round to the nearest two decimal points there. Otherwise, the precision of the decimal number is simply ridiculous and only serves to clutter the screen.